src/shared/AppModal.jsx/AppModal.jsx

```diff
@@ -0,0 +1,36 @@
+import { AiOutlineCloseCircle } from 'react-icons/ai';
```

Comment:
AppModal.jsx folder name:) Misprint?

Reply:
Experiencing the consequences of both working and coding simultaneously))